Dockerfile

```diff
@@ -5,11 +5,11 @@ WORKDIR /go/src/github.com/gardener/logging
 COPY . .
 
 RUN  make plugin
-#############      fluent-bit       #############
-FROM fluent/fluent-bit:1.5.4 AS fluent-bit
+#############      carrier       #############
+FROM alpine:3.12.0 AS carrier
 
-COPY --from=builder /go/src/github.com/gardener/logging/build /fluent-bit/plugins
+COPY --from=builder /go/src/github.com/gardener/logging/build /source/plugins
 
 WORKDIR /
 
-ENTRYPOINT ["/fluent-bit/bin/fluent-bit"]
\ No newline at end of file
+ENTRYPOINT ["cp /source/plugins/. /plugins -fr"]
```

Comment:
I have no strong opinion, but I think you can completely remove the entry point. It is designed for daemon like process while `cp` is not such.

Reply:
ok